src/main.rs

```diff
@@ -63,7 +73,17 @@ fn main() -> Result<()> {
                 }
             };
             let env = matches.value_of("env").map(|v| v.to_owned());
-            println!("{}", reqq.execute(req, env)?);
+            let req_args = matches.values_of("arg").unwrap();
+            let mut extra_args: HashMap<&str, &str> = HashMap::new();
+            for arg in req_args {
+                let kv_pair: Vec<&str> = arg.splitn(2, "=").collect();
+                // I'm sure there is a better Rust way to do this.
+                // If the arg key/value pair is malformed this should
+                // gracefully fail. Not sure how to best to protect against
+                // the failure.
+                extra_args.insert(kv_pair[0], kv_pair[1]);
```

Comment:
```suggestion
                extra_args.insert(kv_pair[0], serde_json::from_str(kv_pair[1])?);
```

Once all the `HashMaps` are updated to be `HashMap<&str, serde_json::value::Value>` this will need to convert from the `&str` input to the `Value` type.